ee/codegen/src/__test__/__snapshots__/project.test.ts.snap

```diff
@@ -1,7 +1,8 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
 exports[`WorkflowProjectGenerator > inlude sandbox > should include a sandbox.py file when passed sandboxInputs 1`] = `
-"from vellum.workflows.sandbox import SandboxRunner
+"from vellum import ARRAYRequest, StringChatMessageContentRequest, STRINGRequest
```

Comment:
Where is `ARRAYRequest` and `STRINGRequest` coming from? Why is `ChatMessageRequest` missing?

Reply:
This looks like a bug in the logic here: https://github.com/vellum-ai/vellum-python-sdks/blob/175711d9ef4bc8579afff035aa485150b30573f5/ee/codegen/src/generators/chat-message-content.ts#L52